xwf/gateway/deploy/roles/pkgrepo/tasks/redhat.yml

```diff
@@ -0,0 +1,36 @@
+---
+################################################################################
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+################################################################################
+
+- name: Add GPG key for magma repository
+  rpm_key:
+    key: https://facebookconnectivity.jfrog.io/artifactory/{{ repo }}/{{ distribution }}/7/repodata/repomd.xml.key
+    state: present
+
+- name: Add JFrog key
```

Comment:
Can be removed when you changed the way to get the pub key

Reply:
Done.